evaluate.py

```diff
@@ -0,0 +1,99 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+
+from pyro.contrib.forecast.evaluate import backtest, eval_crps, eval_mae, logger
+from pyro.ops.stats import quantile
+
+from util import M5Data
+
+
+@torch.no_grad()
+def eval_rmse(pred, truth):
+    """
+    Like pyro.contrib.forecast.eval_rmae but does not average over batch dimensions.
+    """
+    pred = pred.mean(0)
+    error = pred - truth
+    return (error * error).reshape(truth.shape[:-2] + (-1,)).mean(-1).sqrt()
+
+
+@torch.no_grad()
+def eval_pl(pred, truth):
+    """
+    Computes pinball loss over 9 quantiles 0.005, 0.165, 0.25, 0.5, 0.75, 0.835, 0.975, 0.995.
+    """
+    us = torch.tensor(M5Data.quantiles, dtype=pred.dtype, device=pred.device)
+    pred = quantile(pred, probs=us, dim=0)  # 9 x batch_shape x duration x D
```

Comment:
Subtle point: I believe the `quantile` function is actually biased and we should fix it in Pyro. That's a separate discussion; I'll open a PR for discussion.